### DISCUSSION TITLE
Strange messages upgrading WSL-Ubuntu

### DISCUSSION THREAD

**Discussion by angelog0** [microsoft/WSL#9762] [Q&A]
### Windows Version

Microsoft Windows [Versione 10.0.19045.2673]

### WSL Version

Versione WSL: 1.1.3.0 Versione kernel: 5.15.90.1 Versione WSLg: 1.0.49 Versione MSRDC: 1.2.3770 Versione Direct3D: 1.608.2-61064218 Versione DXCore: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp versione Windows: 10.0.19045.2673

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

Linux version 5.15.90.1-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Fri Jan 27 02:56:13 UTC 2023

### Distro Version

Release:        22.04

### Other Software

.

### Repro Steps

```
$ sudo apt upgrade
Lettura elenco dei pacchetti... Fatto
Generazione albero delle dipendenze... Fatto
Lettura informazioni sullo stato... Fatto
Calcolo dell'aggiornamento... Fatto
Get more security updates through Ubuntu Pro with 'esm-apps' enabled:
  libmagickcore-6.q16-dev libmagickwand-dev imagemagick libopusfile0
  libopenexr-dev libopenexr25 libmagickcore-dev libmagickcore-6.q16-6-extra
  libmagickwand-6.q16-6 imagemagick-6.q16 libmagickcore-6.q16-6
  imagemagick-6-common libmagickcore-6-arch-config libmagickwand-6-headers
  libmagickwand-6.q16-dev libmagickcore-6-headers
Learn more about Ubuntu Pro at https://ubuntu.com/pro
#
# "Phased updates" allow careful rollout of deb package changes at scale.
# You may see these as packages that have been "kept back". Learn more at
# https://ubuntu.com/server/docs/about-apt-upgrade-and-phased-updates
#
0 aggiornati, 0 installati, 0 da rimuovere e 0 non aggiornati.
```

What does that mean? Until a few days/weeks ago, this didn't happen..

**Comment by elsaco** [comment]
`Nulla di cui preoccuparsi` ...it's just Canonical nagging you to signup for `Ubuntu Pro`.  More info at https://ubuntu.com/pro

**Reply by angelog0** [reply]
Maybe is time to adopt another distribution... its a pity that it spam the users :-(
